src/Bird.java

```diff
@@ -35,17 +35,23 @@ public Bird() {
     }
 
     public void update() {
-        yvel += gravity;
```

Comment:
If we want to prevent the bird from falling after the game is over, we can add the following on line #38. It accomplishes the same thing we're doing here but with less code.

```
if (this.dead)
  return;
```

Reply:
I tested this option. If we use this method, then the bird freezes in the air.
![screen1](https://user-images.githubusercontent.com/36668134/76797506-bed12680-67de-11ea-8a0b-18dc614818e0.PNG)

Reply:
If we use my version, the bird will gradually fall to the ground.
![screen2](https://user-images.githubusercontent.com/36668134/76797598-f17b1f00-67de-11ea-99c6-3b6f7b596957.PNG)